bindings/rust/src/lib.rs

```diff
@@ -2,10 +2,15 @@
 // Copyright 2019-2020 The Fizzy Authors.
 // SPDX-License-Identifier: Apache-2.0
 
+//! This is a Rust interface to [Fizzy](https://github.com/wasmx/fizzy), a WebAssembly virtual machine.
```

Comment:
What's the difference between `//!` and `///`?

Reply:
`//!` is the description shown on the top of docs.rust.rs, while `///` is the doxygen compatible description for syntactical elements.